caravel/assets/javascripts/explorev2/components/TextField.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import { FormGroup, FormControl } from 'react-bootstrap';
+import ControlLabelWithTooltip from './ControlLabelWithTooltip';
+
+const propTypes = {
+  label: PropTypes.string,
+  description: PropTypes.string,
+};
+
+const defaultProps = {
+  label: null,
+  description: null,
+};
+
+export default function TextField({ label, description }) {
+  return (
+    <FormGroup controlId="formInlineName">
```

Comment:
many of the `*Field` components have a couple adjacent elements so it's kind of nice to keep the `<FieldGroup>` wrapper in each one so we don't have to add an extra `<div>` around the adjacent elements.